composer.json

```diff
@@ -27,7 +27,6 @@
         "drupal/field_group": "~1.0-rc6",
         "drupal/file_url": "~1.0",
         "drupal/flag": "^4.0@alpha",
-        "drupal/geocoder": "dev-3.x",
```

Comment:
This shouldn't be removed. We depend directly on the Drupal Geocoder module.

Reply:
In fact we depend on `oe_webtools_location` which depends on `geocoder`.